docs/design/router.md

```diff
@@ -137,6 +138,15 @@ Allows defining arbitrary routing rules using a JavaScript function. The functio
 
 > JavaScript-based functionality is disabled by default. Please refer to the Druid [JavaScript programming guide](../development/javascript.md) for guidelines about using Druid's JavaScript functionality, including instructions on how to enable it.
 
+### Routing of SQL queries
```

Comment:
do you want to add a note here about JDBC queries?

Reply:
Added.